### ISSUE TITLE
How is the game state data synchronized with media?

### ISSUE THREAD

**Issue by murillo128** [fluffy/draft-jennings-game-state-over-rtp#24]
Is rapid synchronization required or just normal rtcp recevier reports? Do we need to add a new sdp group type as the LS one in rfc5888?

**Comment by fluffy** [comment]
Good point. The way we are using it right now, it does not matter if it is out of sync by 20 ms but if you doing something like transferring the parameters for a facial model of head that was talking, you would want that level of sync.  

I had not though about this much but it seems you could just use LS groups that tied the m lines for the gamestate to the audio. You see any problem with just doing that ?

**Comment by suhasHere** [comment]
LS should be good here .. we might also want to define categories when bundled .

**Comment by murillo128** [comment]
Yes, LS would be enough, would be good also to specify how to set/use the cname and SDES items.
